stabilizer/src/main/java/com/hazelcast/stabilizer/worker/tasks/AbstractWorkerTask.java

```diff
@@ -56,8 +60,24 @@ public long getOperationCount() {
         return operationCount.get();
     }
 
+    /**
+     * Override this method if you need to execute code before {@link #run()} is called.
+     */
+    protected void beforeRun() {
+    }
+
     protected abstract void doRun(O operation);
 
+    /**
+     * Override this method if you need to execute code after {@link #run()} is called.
+     */
+    protected void afterRun() {
+    }
+
+    protected int nextInt() {
```

Comment:
I would call this 'randomInt'. Next int doesn't reveal the intent. randomInt does.